client/interfaces.go

```diff
@@ -226,6 +226,13 @@ type InstanceServer interface {
 	RenameNetwork(name string, network api.NetworkPost) (err error)
 	DeleteNetwork(name string) (err error)
 
+	// Network forward functions ("network_forward" API extension)
+	GetNetworkForwards(networkName string) ([]api.NetworkForward, error)
```

Comment:
We probably should have a `GetNetworkForwardAddresses(networkName string) ([]string, error)` to line up with our usual pattern

Reply:
Yep I've added.